packages/suite/src/components/suite/modals/ReduxModal/UserContextModal/AddAccountModal/AddAccountButton/AddAccountButton.tsx

```diff
@@ -33,73 +28,54 @@ const verifyAvailability = ({
     if (unavailableCapability === 'no-capability') {
         return 'TR_ACCOUNT_TYPE_NO_CAPABILITY';
     }
-    if (!account) {
-        // discovery failed?
-        return 'MODAL_ADD_ACCOUNT_NO_ACCOUNT';
-    }
-    if (emptyAccounts.length === 0) {
-        return 'MODAL_ADD_ACCOUNT_NO_EMPTY_ACCOUNT';
-    }
-    if (emptyAccounts.length > 1) {
-        // prev account is empty, do not add another
-        return 'MODAL_ADD_ACCOUNT_PREVIOUS_EMPTY';
-    }
-    if (account.index === 0 && account.empty && account.accountType === 'normal') {
-        // current (first normal) account is empty, do not add another
-        return 'MODAL_ADD_ACCOUNT_PREVIOUS_EMPTY';
-    }
```

Comment:
Those three conditions should not be used only for `EVM`. Maybe also for the one above so that Cardano is fixed and we do not need to spend time on it